com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -546,40 +551,97 @@ public int additionalLightsShadowmapResolution
             get { return (int)m_AdditionalLightsShadowmapResolution; }
         }
 
+        /// <summary>
+        /// Controls the maximum distance at which shadows are visible.
+        /// </summary>
         public float shadowDistance
         {
             get { return m_ShadowDistance; }
             set { m_ShadowDistance = Mathf.Max(0.0f, value); }
         }
 
+        // MTT this is obsolete. What todo?
         public ShadowCascadesOption shadowCascadeOption
         {
             get { return m_ShadowCascades; }
             set { m_ShadowCascades = value; }
         }
 
+        public int cascadeShadowSplitCount
```

Comment:
Does this need to be public, if so add docs.

Reply:
done